nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc

```diff
@@ -614,7 +614,8 @@ Describe a pointer or reference data type. +
 {result_type}  +
  +
 'Base Type' is the '<id>' of a debugging instruction that represents the pointee
- type. +
+ type. If the 'Base Type' is not defined or not available in source program,
+ this operand must refer to <<DebugInfoNone,*DebugInfoNone*>>. + 
```

Comment:
I understand that's your picked solution that is also implemented in the SPIR-V backend but I'd like to understand how this is preferable to the `DebugTypeBasic` trick as suggested in the thread https://github.com/KhronosGroup/SPIRV-Registry/issues/280#issuecomment-2393436571

I think DebugTypeBasic with `FlagUnknownPhysicalLayout` could be a suitable solution.

Reply:
@Naghasan I believe that answer https://github.com/KhronosGroup/SPIRV-Registry/issues/280#issuecomment-2393712365 addresses your question. I'm open to further discussion.